### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by bartfeenstra** [LionsAd/drupal_ti#41]
Thanks a lot for publishing this code! The more I read through it, the more useful features I find.

**Comment by sylus** [comment]
Agreed! Really appreciate the workflow and the fact that it supports Travis' container infrastructure. The speedup on the behat tests alone was impressive nearly half the time.

**Comment by vever001** [comment]
Just wanted to thank you all as well for your work, very nice tool!
I just opened a [PR](https://github.com/mkalkbrenner/search_api_solr/pull/34) to integrate it in search_api_solr 8.x-3.x which make the whole travis script much more organized.
